util/src/main/java/io/kubernetes/client/informer/cache/ReflectorRunnable.java

```diff
@@ -160,6 +163,8 @@ public void run() {
           closeWatch();
         }
       }
+    } catch (WatchExpiredException e) {
```

Comment:
It looks like this code block is unreachable?
`WatchExpiredException` is thrown at line 134 and then caught by line 135, but is never re-thrown.
It hits the `return;` at line 161, which fixes the bug. However, this one looks unused, unless the `exceptionHandler` re-throws the exception.
Please correct me in case I'm wrong.

Reply:
@haoming-db oh oversight! you're right, the `catch (WatchExpiredException e)` block isnt actually taking effect. but in the `catch(Throwable)` block the thrown `WatchExpiredException` will be finally reaching the `return` statement there. i just opened https://github.com/kubernetes-client/java/pull/1861 and picked it to this pull together. thanks!